editor/components/post-switch-to-draft-button/style.scss

```diff
@@ -0,0 +1,5 @@
+.editor-post-publish-dropdown__publish-button-container {
+	.editor-post-publish-dropdown__switch-to-draft {
```

Comment:
Is the nesting necessary?

Reply:
The nesting _per se_ isn't, but the added specificity is.

Reply:
Yes, needed the core-ui thing otherwise :(